presto-main/src/main/java/io/prestosql/sql/planner/iterative/rule/ExpressionRewriteRuleSet.java

```diff
@@ -65,8 +63,7 @@ public ExpressionRewriteRuleSet(ExpressionRewriter rewriter)
                 aggregationExpressionRewrite(),
                 filterExpressionRewrite(),
                 joinExpressionRewrite(),
-                valuesExpressionRewrite(),
-                applyExpressionRewrite());
+                valuesExpressionRewrite());
     }
```

Comment:
You could update `DesugarAtTimeZone` and `DesugarRowSubscript` which currently override `ExpressionRewriteRuleSet.rules()` to exclude `applyExpressionRewrite()`.